frontend/pages/review.tsx

```diff
@@ -35,29 +37,86 @@ function ErrorWrapper({ message }: { message: string | undefined }) {
 }
 
 function ApprovalList({ category, filter }: { category: ApprovalCategory; filter: ApprovalFilterType }) {
+  const { approvals, isApprovalsLoading, isApprovalsError, mutateApprovals } = useListApprovals(category, filter)
```

Comment:
`approvals` can be `undefined` here due to our implementation in `useListApprovals`. We should update line 25 inside that hook (see `frontend/data/approvals.ts`) from:
```
approvals: data?.approvals
```
to
```
approvals: data ? data.approvals : []
```

That way we can guarantee that we're dealing with an array, instead of having to check for `undefined`.